instance_control/readme.md

```diff
@@ -28,15 +37,15 @@ bubuku_cluster.py terminate \
 ```
 
 ### Attach
-
+The command creates a new instance and attaches provided volume to that instance.
```

Comment:
Maybe in the future we can improve it in a way that you don't have to specify the volume-id but the bubuku-script discovers the first unattached volume and attaches it automatically. (For example all volumes can have some special names like bubuku-live-volume-5a56bc65caa1208 so that it will be easy to discover them)